privacy-policy.html

```diff
@@ -0,0 +1,243 @@
+  <link href="https://fonts.googleapis.com/css?family=Lato:100,300,300i,400&display=swap"
+        rel="stylesheet">
+  <link rel="icon" type="image/png" href="/public/images/favicon-bird-gold.jpg">
+  <script type="module" src="https://unpkg.com/ionicons@5.0.0/dist/ionicons/ionicons.esm.js"></script>
+  <script nomodule="" src="https://unpkg.com/ionicons@5.0.0/dist/ionicons/ionicons.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <title>Privacy Policy</title>
+</head>
+<body>
+<header class="js--section-top header--white">
+  <nav>
+    <div class="row">
+      <img alt="SkyPilot logo" src="public/images/SkyPilot-logo-gold-v2.svg" class="logo">
+      <a class="wordmark js--scroll-to-top" href="index.html">SkyPilot</a>
+      <ul class="main-nav">
+        <li><a href="index.html#anchor-services">About</a></li>
+        <li><a href="index.html#anchor-portfolio">Portfolio</a></li>
+        <li><a href="index.html#anchor-contact">Contact</a></li>
+      </ul>
+    </div>
+  </nav>
+</header>
+
+<div class="page-copy legal">
+  <h6>Last updated: March 23, 2020</h6>
+  <h1>Privacy Policy</h1>
+  <p>Please read these terms and conditions carefully before using Our Service.</p>
+  <p>This Privacy Policy describes Our policies and procedures on the collection, use and disclosure of Your information when You use the Service and tells You about Your privacy rights and how the law protects You.</p>
+  <p>We use Your Personal data to provide and improve the Service. By using the Service, You agree to the collection and use of information in accordance with this Privacy Policy.</p>
+
+  <h2>Interpretation and Definitions</h2>
+  <h3>Interpretation</h3>
+  <p>The words of which the initial letter is capitalized have meanings defined under the following conditions. The following definitions shall have the same meaning regardless of whether they appear in singular or in plural.</p>
+
+  <h3>Definitions</h3>
+  <p>For the purposes of this Privacy Policy:</p>
+  <ul>
+    <li><strong>You</strong> means the individual accessing or using the Service, or the company, or other legal entity on behalf of which such individual is accessing or using the Service, as applicable.</li>
+    <li><strong>Company</strong> (referred to as either "the Company", "We", "Us" or "Our" in this Agreement) refers to SkyPilot Cloud Consulting, in Fernie, BC, Canada.</li>
+    <li><strong>Affiliate</strong> means an entity that controls, is controlled by or is under common control with a party, where "control" means ownership of 50% or more of the shares, equity interest or other securities entitled to vote for election of directors or other managing authority.</li>
+    <li><strong>Website</strong> refers to SkyPilot, accessible from https://www.skypilot.dev</li>        <li><strong>Service</strong> refers to the Website.</li>
+    <li><strong>Country</strong> refers to: British Columbia, Canada</li>
```

Comment:
```suggestion
    <li><strong>Country</strong> refers to British Columbia, Canada</li>
```
Either
- change to "Country refers to Canada" or
- change to "Jurisdiction refers to British Columbia, Canada" and replace "Country" with "Jurisdiction" elsewhere in the document